vsintegration/tests/FSharp.Editor.Tests/CodeFixes/ConvertCSharpLambdaToFSharpLambdaTests.fs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+module FSharp.Editor.Tests.CodeFixes.ConvertCSharpLambdaToFSharpLambdaTests
+
+open Microsoft.VisualStudio.FSharp.Editor
+open Xunit
+
+open CodeFixTestFramework
+
+let private codeFix = ConvertCSharpLambdaToFSharpLambdaCodeFixProvider()
+
+let private diagnostic = 0039 // Something is not defined
+
+[<Fact>]
+let ``Fixes FS0039 for lambdas`` () =
+    let code =
+        """
+let id = x => x
+"""
+
+    let expected =
+        Some
+            {
+                Message = "Use F# lambda syntax"
+                FixedCode =
+                    """
+let id = fun x -> x
```

Comment:
Maybe not the best example, in this case we should probably change it to
```fsharp
let id x = x
```

But I suppose there could be another code fix for removing `fun` from `let` bindings which would be triggered afterwards.

Reply:
Nono good point, let's have a more meaningful example here.